identity/webapp/src/routes/assets/index.html.ts

```diff
@@ -0,0 +1,179 @@
+const style = `
+<style>
+  html,
+  body {
+    height: 100vh;
+    margin: 0;
+  }
+
+  body {
+    display: grid;
+    place-items: center;
+    font-family: 'Helvetica Neue Light Web', HelveticaNeue-Light, 'Helvetica Neue Light', 'Helvetica Neue',
+      Helvetica, Arial, sans-serif;
+    font-size: 24px;
+  }
+
+  @media screen and (min-width: 600px) {
+    body {
+      background-color: #f0ede3;
+    }
+  }
+
+  .container {
+    background-color: white;
+  }
+
+  @media screen and (min-width: 600px) {
+    .container {
+      border-radius: 10px;
+      max-width: 40em;
+      padding: 3em 0;
+    }
+  }
+
+  .wrapper {
+    display: grid;
+  }
+
+  @media only screen and (min-width: 600px) {
```

Comment:
I notice this media query has `only` but the others don't. If we're considering old browsers at all, I think this should be added to all the other media queries, or remove it from here - as long as this isn't the one off